### PR TITLE
Friendly choices for editable boolean field.

### PR DESCRIPTION
**Pull request by iurisilvio**
`Yes / No` choices are better than `True / False`.

### CODE REVIEW

File: flask_admin/model/widgets.py [hunk 1]
```diff
@@ -109,7 +109,10 @@ def get_kwargs(self, subfield, kwargs):
         elif subfield.type == 'BooleanField':
             kwargs['data-type'] = 'select'
             # data-source = dropdown options
-            kwargs['data-source'] = {'': 'False', '1': 'True'}
+            kwargs['data-source'] = json.dumps([
+                {'value': '0', 'text': gettext('No')},
```
Comment (pawl) [inline flask_admin/model/widgets.py]: better to use lazy_gettext here, like this: https://github.com/iurisilvio/flask-admin/blob/translate_editable_boolean/flask_admin/model/filters.py#L98

Comment (iurisilvio) [inline flask_admin/model/widgets.py]: The `lu'string'` is not JSON serializable, so the `json.dumps` crashes if I use the `lazy_gettext`, so I have to evaluate the lazy string before serialization.

Comment (pawl) [inline flask_admin/model/widgets.py]: Oops, looks good then.

Any idea why it's using "0" now instead of ""? Does that cause any issues?

Comment (iurisilvio) [inline flask_admin/model/widgets.py]: The `"0"` does not work the way I expected. Going back to the empty string.

**Comment by iurisilvio** [comment]
@pawl I reverted to use empty string instead of `"0"`.

It still has a bug, the current value is not selected. It already exists, because the widget value is an HTML string to render the icon. I don't know yet how to fix this.

**Pull request merged**